@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fn_getDeltaTime.sqf

```diff
@@ -19,7 +19,7 @@ Author:
 if (isGamePaused) exitWith { fza_sfmplus_deltaTime = 0.0; };
 
 fza_sfmplus_currentTime    = diag_tickTime;
-fza_sfmplus_deltaTime      = (fza_sfmplus_currentTime - fza_sfmplus_previousTime);
+fza_sfmplus_deltaTime      = [fza_sfmplus_deltaTime_avg, (fza_sfmplus_currentTime - fza_sfmplus_previousTime)] call fza_sfmplus_fnc_getSmoothAverage;
```

Comment:
Do you really not want the correct delta time? Could this cause issues with lag spikes?

Reply:
i want this time to be as smooth as possible, the deltaTime call prevents the deltaTime value from getting to large, the averaging will keep it smooth overall.